docs/reference/buffer_format.rst

```diff
@@ -105,23 +105,29 @@ Example buffer formats
 ......................
 
 ``"2f"`` has a count of ``2`` and a type of ``f`` (float). Hence it describes
-two floats. The size of the floats is unspecified, so it defaults to ``4``. The
-usage of the buffer is unspecified, so defaults to ``/v`` (vertex), meaning
-each successive pair of floats in the array are passed to successive vertices
-during the render call.
+two floats, passed to a ``vec2`` shader variable. The size of the
```

Comment:
"attribute" is more accurate. "variable" is not consistent with GLSL terms. I think there are couple of them in the doc. We also refer to shaders as "programs" in monderngl, but I guess since attributes are present in the vertex shader it's probably still accurate to use "shader".